assets/sass/vendor/_mdc-button.scss

```diff
@@ -89,6 +89,11 @@
 		}
 	}
 
+	&--surfaces {
+
+		@include mdc-button-filled-accessible($c-surfaces-on-background);
```

Comment:
Let's use `$c-surfaces-inverse-surface` here – the value is actually the same but this better matches the token in the design 👍 

<img width="645" alt="image" src="https://user-images.githubusercontent.com/1621608/181079989-a624afb2-f024-4645-876a-80962f8ab6c3.png">

Reply:
Updated, thank you!